### PR TITLE
media-plugins/gst-transcoder: Fix libdir for meson build

### PR DESCRIPTION
**Pull request by jok-ts**
dev-util/meson guesses libdir for the build incorrectly if /usr/bin/dpkg-architecture exists, see 
Gentoo-Bug: 591564

Use the known correct value for Gentoo instead of the default guess.

### CODE REVIEW

**Comment by jok-ts** [comment]
Some justification for the package versions in this PR:

If someone has already successfully built pitivi-0.97.1 there should be no need for a rebuild even if the gst-transcoder as a dependency gets upgraded from 1.8.[12] to 1.8.2-r1, so there's no revbump needed there. A successful build indicates that the existing gst-transcoder was already correctly installed.

For gst-transcoder-1.8.2 alone, there's no guarantee that the installed files are at the correct location so a revbump is needed to ensure this.

File: media-plugins/gst-transcoder/gst-transcoder-1.8.2-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+inherit python-any-r1 xdg
+
+DESCRIPTION="GStreamer Transcoding API"
+HOMEPAGE="https://github.com/pitivi/gst-transcoder"
+SRC_URI="https://github.com/pitivi/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (SoapGentoo) [inline media-plugins/gst-transcoder/gst-transcoder-1.8.2-r1.ebuild]: don't commit ebuilds with stable KEYWORDS

Comment (jok-ts) [inline media-plugins/gst-transcoder/gst-transcoder-1.8.2-r1.ebuild]: Right, fixed.

**Comment by SoapGentoo** [comment]
@jok-ts please squash the last commit into your main edit, as 1) it doesn't have the right COMMITMSG and 2) doesn't need to live on its own

**Comment by monsieurp** [comment]
@EvaSDK @leio

**Comment by EvaSDK** [comment]
Looks like fine work, :+1: to merge.

File: media-plugins/gst-transcoder/gst-transcoder-1.8.2-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	${PYTHON_DEPS}
+	>=dev-util/meson-0.28.0
+	virtual/pkgconfig
+"
+
+src_configure() {
+	# Not a normal configure
+	# --buildtype=plain needed for honoring CFLAGS/CXXFLAGS and not
+	# defaulting to debug
+	./configure --prefix=/usr --libdir="$(get_libdir)" --buildtype=plain || die
```
Comment (SoapGentoo) [inline media-plugins/gst-transcoder/gst-transcoder-1.8.2-r1.ebuild]: could you make `--prefix="${EPREFIX}/usr"` instead? We should always aim for Prefix-awareness

File: media-video/pitivi/pitivi-0.97.1.ebuild [hunk 3]
```diff
@@ -82,6 +82,7 @@ src_configure() {
 	./configure \
 		--prefix=/usr \
```
Comment (SoapGentoo) [inline media-video/pitivi/pitivi-0.97.1.ebuild]: same here with `EPREFIX`

**Comment by jok-ts** [comment]
@EvaSDK @leio who would be the one to actually pull this one?

**Pull request closed without merging**